mappings/net/minecraft/server/config/ServerConfigurationManager.mapping

```diff
@@ -45,8 +45,13 @@ CLASS wg net/minecraft/server/config/ServerConfigurationManager
 	FIELD v latencyUpdateTimer I
 	METHOD a reloadWhitelist ()V
 	METHOD a setViewDistance (I)V
+	METHOD a sendToAllPlayers (Lapv;Liz;)V
 	METHOD a setGameMode (Lazv;)V
 	METHOD a addToOperators (Lcom/mojang/authlib/GameProfile;)V
+	METHOD a onPlayerConnect (Lin;Luf;)V
+		ARG 1 connection
+		ARG 2 player
+	METHOD a onChatmessage (Liz;Z)V
```

Comment:
onChatMessage, and I assume you've mapped the relevant packets if any?

Reply:
the packets were mapped already, thats how i traversed this and found this..
i am trying to only map stuff i am 100% certain about

to complete all necessary hooks